src/components/issues.js

```diff
@@ -91,10 +92,13 @@ class Issues {
   fetchAndLoadOpenIssues() {
     this.adapter
       .getOpenIssues()
-      .then(issues => {
-        issues.forEach(issue => this.openIssuesArray.push(new Issue(issue)))
```

Comment:
This type of `forEach` to `push` into an empty array ends up being the same as a `map` operation (see below)